uber/site_sections/registration.py

```diff
@@ -165,15 +164,9 @@ def qrcode_generator(self, data):
         this function.  Or, offload image generation to a dedicated microservice that replicates this functionality.
 
         """
-        if not c.QR_CODE_PASSWORD or not qr_cipher:
-            # Don't generate a 2D barcode if we aren't configured for encryption
-            return '2D barcode password not set, cannot generate images'
-
-        encrypted_data = c.EVENT_QR_ID + bytes.decode(binascii.hexlify(qr_cipher.encrypt(uuid.UUID(data).hex)))
-
         checkin_barcode = treepoem.generate_barcode(
             barcode_type='azteccode',
-            data=encrypted_data,
+            data=c.EVENT_QR_ID + str(data),
```

Comment:
> UUIDs are designed to have very low frequency of collision

Saying "low frequency of collision" kind of understates it, and we could instead say, "there are 21267647932558653966460912964485513216 possible uuid4 values, so every computer ever built could generate as many as possible every second from now until the heat death of the universe without ever seeing a duplicate".

With that being said, I have no objection to prepending the event id, since that might be useful in a hypothetical application which scanned barcodes from multiple events, or in helping us emit more useful debug messages, etc.  I just don't find "reduce collisions" to be a justifiable reason :)